azure.key

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEzxfKqNeUnhJGkFKo8uE3GRoCmfK/
```

Comment:
this is expected?

Reply:
Whoops, that was from debugging the azure kms bug. No idea how it ended up in this branch!